### PR TITLE
Test rust 1.57 on CI

### PR DESCRIPTION
**Pull request by kchibisov**

### CODE REVIEW

File: glutin/src/context.rs [hunk 1]
```diff
@@ -77,6 +77,7 @@ impl Context<PossiblyCurrent> {
     }
 }
 
+#[allow(rustdoc::broken_intra_doc_links)]
```
Comment (MarijnS95) [inline glutin/src/context.rs:80]: Why is this? Can't we fix the links? :disappointed:

Comment (kchibisov) [reply glutin/src/context.rs:80]: The rust 1.57 bugs here. The stable passes for example.

Comment (MarijnS95) [reply glutin/src/context.rs:80]: That's ugly, means mistakes won't be caught on `stable` either.

Comment (MarijnS95) [reply glutin/src/context.rs:80]: Why anyway do we build `rustdoc` on the MSRV CI step?

Comment (kchibisov) [reply glutin/src/context.rs:80]: Well, given that's the latest release you see this code... I'm actually not sure why are we using the msrv for docs. Might run those on stable.

**Pull request merged**